libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```
@@ -16,7 +16,81 @@
     "overview": {
       "breadcrumb": {
         "meteringPoints": "Metering points"
+      },
+      "meteringPointType": "Type of Metering Point",
+      "meteringMethod": "Metering Method",
+      "gsrnNumber": "Metering Point ID",
+      "connectionState": "Connection State",
+      "settlementMethod": "Settlement Method",
+      "readingOccurrence": "Meter Reading Periodicity",
+      "meteringPointNotFound": {
+        "title": "Vi fandt desværre intet målepunkt [EN]",
+        "message": "Tjek for eventuelle tastefejl og prøv igen eller gå til søgning. [EN]",
+        "button": "Gå til søgning [EN]"
```

Maybe Jonas has a good phrase for this one? Otherwise I would just translate it. 
```suggestion
        "button": "Go to search"
```